dev-client/src/screens/ProjectTeamScreen/components/UserItem.tsx

```diff
@@ -0,0 +1,154 @@
+/*
+ * Copyright © 2023 Technology Matters
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+import {Button, Center, Image, Pressable} from 'native-base';
+import {User} from 'terraso-client-shared/account/accountSlice';
+import {useTranslation} from 'react-i18next';
+import {useMemo} from 'react';
+import {ProjectMembership} from 'terraso-client-shared/project/projectSlice';
+import {ConfirmModal} from 'terraso-mobile-client/components/modals/ConfirmModal';
+import {formatName} from 'terraso-mobile-client/util';
+import {
+  Box,
+  HStack,
+  VStack,
+  Badge,
+  Text,
+} from 'terraso-mobile-client/components/NativeBaseAdapters';
+
+type TriggerProps = {
+  onOpen: () => void;
+  message: string;
+};
+
+function LeaveProjectTrigger({onOpen, message}: TriggerProps) {
+  return (
+    <Center>
+      <Button
+        size="sm"
+        my={2}
+        w="50%"
+        _text={{color: 'error.main'}}
+        bgColor="grey.200"
+        onPress={onOpen}>
+        {message}
+      </Button>
+    </Center>
+  );
+}
+
+type InfoProps = {
+  membership: ProjectMembership;
+  user: User;
+  isCurrentUser: boolean;
+};
+
+const UserInfo = ({membership, user, isCurrentUser}: InfoProps) => {
+  const {t} = useTranslation();
+  const userLabel = useMemo(() => {
+    let label = formatName(user.firstName, user.lastName);
+
+    if (isCurrentUser) {
+      label += ` (${t('general.you')})`;
+    }
+    return label;
+  }, [user, isCurrentUser, t]);
+
+  return (
+    <HStack space={3} justifyContent="space-between" alignItems="center">
+      <Box>
+        <Image
+          variant="profilePic"
+          source={{uri: user.profileImage}}
+          alt="profile pic"
+        />
+      </Box>
+      <Text flex={3}>{userLabel}</Text>
+      <Box>
+        <Badge
+          variant="chip"
+          bg="primary.lighter"
+          py="5px"
+          px="10px"
+          _text={{color: 'text.primary'}}>
+          {t('general.role.' + membership.userRole)}
+        </Badge>
+      </Box>
+    </HStack>
+  );
+};
+
+type ItemProps = {
+  membership: ProjectMembership;
+  user: User;
+  isCurrentUser: boolean;
+  isManager: boolean;
+  hasSingleManager: boolean;
+  removeUser: () => void;
+  memberAction: () => void;
+};
+
+export const UserItem = ({
+  membership,
+  user,
+  isCurrentUser,
```

Comment:
question: what does this indicate? Does it mean the user is a member of the project?

Reply:
This indicates whether the user in question is the currently logged-in user. The language was carried forward from a prior implementation and it makes sense to me, but I am happy to change if you have a more clear suggestion.